src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -2,8 +2,12 @@
 @use "scss/attendee/space";
 @use "scss/attendee/font";
 @use "scss/attendee/layout";
+@use "scss/attendee/layer";
 @use "scss/attendee/breakpoint";
 
+$-z-index--space: layer.above(0);
+$-z-index--ui: layer.above($-z-index--space);
```

Comment:
Same principle as above, different module. This module has it's own relative `z-index` values.